next.config.js

```diff
@@ -1,12 +1,15 @@
-module.exports = {
-  env: {
-    FIREBASE_API_KEY: process.env.FIREBASE_API_KEY,
-    FIREBASE_AUTH_DOMAIN: process.env.FIREBASE_API_KEY,
-    FIREBASE_DATABASE_URL: process.env.FIREBASE_DATABASE_URL,
-    FIREBASE_PROJECT_ID: process.env.FIREBASE_PROJECT_ID,
-    FIREBASE_STORAGE_BUCKET: process.env.FIREBASE_STORAGE_BUCKET,
-    FIREBASE_APP_ID:  process.env.FIREBASE_APP_ID,
-    FIREBASE_MESSAGING_SENDER_ID: process.env.FIREBASE_MESSAGING_SENDER_ID,
-    RECAPTCHA_SITE_KEY:  process.env.RECAPTCHA_SITE_KEY,
-  },
+module.exports = () => {
+  if (process.env.DEPLOY_ENV !== "PRODUCTION") return {}
+  return {
+    env: {
+      NEXT_PUBLIC_FIREBASE_API_KEY: "AIzaSyDGa7alU0NhfBATSQ6CalkY4Za9wWPrM7o",
+      NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN: "nwplus-ubc.firebaseapp.com",
+      NEXT_PUBLIC_FIREBASE_DATABASE_URL: "https://nwplus-ubc.firebaseio.com",
+      NEXT_PUBLIC_FIREBASE_PROJECT_ID: "nwplus-ubc",
+      NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET: "nwplus-ubc.appspot.com",
+      NEXT_PUBLIC_FIREBASE_MEASUREMENT_ID: "G-BT2W7DSL1G",
+      NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID: "306881258768",
+      NEXT_PUBLIC_FIREBASE_APP_ID: "1:306881258768:web:bc922148732abee79f7195"
```

Comment:
Do these need to be hard-coded?

Reply:
I put the keys there so when someone wants to build the prod version of CMS locally, they can just call yarn build:prod

Reply:
It's cool, these prod values are pretty easy to find even if we try to hide them, so its okay to leave them in the file. I do agree with jenny that hard coding them into the function is not the best idea, could you make a `const env` object at the top of the file so its easier to see and edit those values.